packages/scripts/src/commands/e2e-data/parse.ts

```diff
@@ -0,0 +1,92 @@
+import { Command } from "commander";
+import fs from "fs-extra";
+import path from "path";
+import XLSX from "xlsx";
+import prettier from "prettier";
+import * as mappings from "./mappings";
+import snippets from "./snippets";
+import { ROOT_DIR } from "../../paths";
+
+interface IProgramOptions {
+  input: string;
+  output: string;
+}
+/** [WIP] Helps turn XLSX data into cypress compatible test runs. */
+const program = new Command("parse");
+export default program
+  .description("Parse worksheet into JS!")
+  .requiredOption("-i --input <string>", "path to input xlsx file (relative to project root)")
+  .requiredOption("-o --output <string>", "path to output spec file (relative to project root")
+  .action((options: IProgramOptions) => {
```

Comment:
I made these changes (previously input was an arg and output piped to file via script). The main reasons for doing so are:
1) A bit easier to scale if the list grows (can add type interface and order not important)
2) Piping output makes it harder to add debug logs (would be printed to the file) and can have incompatibilities depending on os and client (`>` operator may or may not exist)

But I did think it was a really need solution to essentially use the logs to write line-by-line. I had to find another workaround instead by storing an array of responses and concatenating at the end